### ISSUE TITLE
Installation difficulties.

### ISSUE THREAD

**Issue by marzman7** [vially/googlemusic-xbmc#1]
I hope I'm commenting in the right place for this issue. I'd like to say thanks for getting an early start on making this addon for Google Music! When I install, I am able to put my credentials, but then the plugin gets an error. It has never loaded my Google Music. Do I have to install Python? I had a version already, but I installed the newest as well. I did that so I could install setuptools. However, setup tools still says I have the wrong version of python. Forgive my simple questions, please. I'm still getting used to XBMC

**Comment by vially** [comment]
Thanks for trying this early version.

You do need to have a version of python installed but most probably you already have one. Depending on the way you installed XBMC python might have been installed automatically as a dependency.
Setuptools is not necessary as long as you make the change mentioned in the known issues section in the README

In order to be able to help you fix the problem I might need a few more things. First of all let me know which OS are you using: linux, windows or mac?

Another thing I need in order to try and fix your error is the [xbmc log file](http://wiki.xbmc.org/index.php?title=Log_file#Location)

The discussion about this add-on on the XBMC forums might also be helpful: http://forum.xbmc.org/showthread.php?tid=134783

Hope this helps!

Cheers!

**Comment by marzman7** [comment]
Thanks for your quick response! I'll be able to reply more when I get home this evening. And I'll check out the forum. 
Mike 27 Then he said, I pray thee therefore, father, that thou wouldest send him to my father's house: 28 For I have five brethren; that he may testify unto them, lest they also come into this place of torment. 29 Abraham saith unto him, They have Moses and the prophets; let them hear them. 30 And he said, Nay, father Abraham: but if one went unto them from the dead, they will repent. 31 And he said unto him, If they hear not Moses and the prophets, neither will they be persuaded, though one rose from the dead. -Luke 16

**Comment by marzman7** [comment]
Thanks! I'm on Windows XP Pro SP3, 32-bit. I've attached my log file.

 
27 Then he said, I pray thee therefore, father, that thou wouldest send him to my father's house: 28 For I have five brethren; that he may testify unto them, lest they also come into this place of torment. 29 Abraham saith unto him, They have Moses and the prophets; let them hear them. 30 And he said, Nay, father Abraham: but if one went unto them from the dead, they will repent. 31 And he said unto him, If they hear not Moses and the prophets, neither will they be persuaded, though one rose from the dead. -Luke 16

**Comment by marzman7** [comment]
Also, I discovered that it's not completely broken... It give me a script error when trying to load all songs. But I tried to load playlists, and it did give me the one playlist I had uploaded. So maybe that will help you understand what's happening.
Thanks again,
Mike
 
27 Then he said, I pray thee therefore, father, that thou wouldest send him to my father's house: 28 For I have five brethren; that he may testify unto them, lest they also come into this place of torment. 29 Abraham saith unto him, They have Moses and the prophets; let them hear them. 30 And he said, Nay, father Abraham: but if one went unto them from the dead, they will repent. 31 And he said unto him, If they hear not Moses and the prophets, neither will they be persuaded, though one rose from the dead. -Luke 16
